document_page/__manifest__.py

```diff
@@ -43,7 +32,5 @@
     'demo': [
         'demo/document_page.xml'
     ],
-    'installable': False,
```

Comment:
Yes, the script now detects if the key is not present and adds it in that case. So removing it is harmless.